Makefile

```diff
@@ -0,0 +1,94 @@
+# Package configuration
+PROJECT = sandbox-ce
+
+# superset upstream configuration
+SUPERSET_REPO = https://github.com/apache/incubator-superset.git
+SUPERSET_VERSION = release--0.32
+SUPERSET_REMOTE = superset
+# directory to sync superset upstream with
+SUPERSET_DIR = superset
+# directory with custom code to copy into SUPERSET_DIR
+PATCH_SOURCE_DIR = srcd
+# name of the superset docker image to build
+SUPERSET_IMAGE_NAME = srcd/superset
```

Comment:
maybe it should be `srcd/sourced-ui` (I don't know)

Reply:
yes, keeping the same name as the repo sounds better.

Reply:
I also created the PR to change it in the compose file https://github.com/src-d/sourced-ce/pull/5